### ISSUE TITLE
Respec errors

### ISSUE THREAD

**Issue by yoavweiss** [WICG/webcomponents#874]
```
Duplicate definition(s) of 'import' at: 1.
Couldn't match "HTMLLinkElement" to anything in the document or in any other document cited in this specification: webidl. See how to cite to resolve the error at: 1.
Couldn't match "Document" to anything in the document or in any other document cited in this specification: webidl. See how to cite to resolve the error at: 1.
Couldn't match "custom element" to anything in the document or in any other document cited in this specification: custom-elements, webidl. See how to cite to resolve the error at: 1.
```

@marcoscaceres - can you take a look?

**Comment by yoavweiss** [comment]
It's particularly problematic that this no-longer-maintained spec breaks. Would be good to find a solution for "freezing" respec specs.

**Comment by marcoscaceres** [comment]
We can bind then to particular ReSpec versions (but they miss out on security updates, etc.). Alternatively, we can “freeze” them by exporting as HTML, and putting the raw ReSpec source file in a different branch.

**Comment by domenic** [comment]
What spec are you referring to? Can you provide a link?

**Comment by marcoscaceres** [comment]
Old import and shadow by the looks of it.

**Comment by domenic** [comment]
They seem readable to me. Fixing these errors wouldn't buy us anything.

**Comment by marcoscaceres** [comment]
True. I’m not sure what we should do here. Are those specs dead? Should we “gut” them?

**Comment by domenic** [comment]
I already gutted as much of shadow DOM as I could. We'll keep working on gutting the rest.

Imports can probably be totally removed, yeah.

**Comment by domenic** [comment]
Or we could just leave them with errors? Is there something I'm missing that makes fixing the errors urgent?

**Comment by yoavweiss** [comment]
Nothing here is urgent - I just stumbled upon them, saw errors and reported them.
The underlying issue is that the current setup for respec specs requires constant maintenance, which is not great for maintained specs, but becomes a real problem for no-longer-maintained ones.
If we can kill this spec, that's not a problem here. But there could be things we want to keep around (for historical purposes) and having their links breaks for no good reason is a shame.

**Comment by yoavweiss** [comment]
Also, apologies for not including a link in my initial report - it's https://w3c.github.io/webcomponents/spec/imports/
